pkgs/tools/text/m2r/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonApplication
+, fetchFromGitHub
+, docutils
+, mistune
+, pygments
+}:
+
+buildPythonApplication rec {
+  pname = "m2r";
+  version = "0.2.1";
+
+  src = fetchFromGitHub {
+    owner = "miyakogi";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-JNLPEXMoiISh4RnKP+Afj9/PJp9Lrx9UYHsfuGAL7uI=";
+  };
+
+  nativeBuildInputs = [
+    docutils
+  ];
+
+  buildInputs = [
```

Comment:
If these need to be available during runtime then they should not be in `buildInputs`.

Reply:
Isn't it the reverse? Things needed only while building are in `nativeBuildInputs` while those needed in run are in`buildInputs`.